Adding ItemSets + Cyberware + Quest Triggers (facts)

VP-EN opened the pull request:
Resolving the issue / proposal / enhancement:
https://github.com/WolvenKit/BraindanceProtocol/issues/3

This should include the "essential" item sets. 
I excluded the ones marked "special" or "quest", since they're mostly non-removable from the players inventory.

Might add as "--dashed" later on, but for now this'll do 👍

Commit:
Add Item Sets

Adding All Item Sets, except the ones marked speciel and/or quest. Reason is that they're mostly non-removable from inventory.

Also updated the README with the new commands and updated the init.lua..

Tested ingame and it works fine.

Commit:
Update itemsets.lua

Commit:
Adding Cyberware

Adding all Cyberware, including the quickhacks.

This excludes the weird quest common hand items, such as "Johnny's Tattoo".

ghost commented:
I ended up also adding all Cyberware items, categorized into the usual quality categories. Also works fine ingame.

Doesn't take account for duplicates though, so it'll add the item, even though you already have one in your inventory. Not sure how to deal with that.

Commit:
Added Ozobs nose + Tattoo cyberware

I added Ozobs nose to the grenade recipes.

I also ended up adding the "tattoo" cyberware (under common cyberware).

Commit:
Added facts.lua

Added a few quest trigger protocols, making the player able to change the outcome of an already completed quest.

Some of these are pretty "spoilery", if you haven't played the game, so I'm not sure what to do about that.

Also, not so sure about how the romance options work. Apparently you have to apply them, each time you reload a savegame.

ghost commented:
Also added facts.lua, with options for changing the outcome of already completed quests / situations.
Some of these are "spoilery", so I'm not sure what to do about that.

osulli reviewed (approved):
Great job.

Also, I don't think the Facts are spoilery - they're vague enough (e.g. X can live or die & Y can be romanced) -- not spoilery IMO.
